wg-api-expression/2020-annual-report.md

```diff
@@ -0,0 +1,40 @@
+# 2020 - WG API Expression - Community Meeting Annual Reports
+
+**What was the initial mission of the group and if it's changed, how?**
+
+Should still be the same as outlined in our [mission statement](https://github.com/kubernetes/community/blob/master/wg-api-expression/README.md), but with a hard focus on Server Side Apply.
+
+**What’s the current roadmap until completion?**
+
+- Server Side Apply (GA targeted for v1.21).
+
+    Currently the WorkGroup is mostly focused on delivering Server Side Apply.
+
+- Our Mission Statement contains various other topics we would like to work on, but finishing and delivering
+a working version of SSA is our primary goal right now.
+
+**Have you produced any artifacts, reports, white papers to date?**
+
+- SSA went to Beta 2 with K8s v1.18
+    
+    This enabled the tracking of managedFields by default on every new resource.
+[There was a Blog Post about it](https://kubernetes.io/blog/2020/04/01/kubernetes-1.18-feature-server-side-apply-beta-2/).
+
+**Is the group active? healthy? contributors from multiple companies and/or end user companies?**
+
+- The group meets actively every two weeks with communication on GitHub and Slack in between.
+- Contributors are from various companies, including Google, IBM, Red Hat and end user companies.
+
+**Is everything in your readme accurate? posting meetings on youtube?**
+
+- The readme should be accurate
+- We’re uploading videos to youtube, but due to some permission problems we send the videos to markyjackson who is kind enough to upload them for us. 
```

Comment:
as long as the videos actually end up posted, and contribx has visibility to this friction, you can probably omit it from the committed report

Reply:
Just as an FYI on this. I double checked access to the playlist and you should have access. Adding videos isn't intuitive however :| You cannot do it from YouTube studio (where you can manage all the videos you upload). Goto the video independently and click on the `save` button under the video in the bottom right. You can then add items to the playlist there.

YouTube Guidelines:
https://www.kubernetes.dev/docs/comms/youtube/

Additional info:
https://support.google.com/youtube/answer/6109639#zippy=%2Cadd-videos

Reply:
Thanks I'll give it a try.